src/custom-hooks/useBroadcastDetails.js

```diff
@@ -115,8 +118,14 @@ export const useBroadcastDetails = ({ _id, username, createdAt, likedBy }) => {
           ).map((option) => ({
             ...option,
             handleClick: isInLoggedUserFollowing
-              ? () => postUnfollowCall(username)
-              : () => postFollowCall(username),
+              ? () => {
+                  dispatch(setFollowUsername(username));
+                  postUnfollowCall(username);
```

Comment:
avoid this type of nesting function calling better to use if else

Reply:
Actually, it was only one level nesting that's why I thought to use the ternary operator here.